packages/preview/sustech-bachelor-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "sustech-bachelor-thesis"
+version = "0.1.0"
+entrypoint = "lib.typ"
```

Comment:
Probably you want 

```suggestion
entrypoint = "sections/lib.typ"
```

Reply:
Really thanks a lot! I'm really confused by the meaning of 'entrypoint'. But after I rebuild my template locally, I rebuild my template in a directly way. I will construct them in the later versions.

Reply:
Really thanks a lot! I'm really confused by the meaning of 'entrypoint'. But after I build my template locally, I move my main files in the folder sections. I will construct them better in the later versions.